packages/vite/src/node/plugins/worker.ts

```diff
@@ -45,31 +46,66 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
         url = await fileToUrl(cleanUrl(id), config, this)
         url = injectQuery(url, WorkerFileId)
       } else {
+        const rollup = require('rollup') as typeof Rollup
```

Comment:
**suggestion (non-blocking):** We may use destructuring here
`const { rollup } = require('rollup') as typeof Rollup`